### DISCUSSION TITLE
Invoice inquiry

### DISCUSSION THREAD

**Discussion by kawamra** [community/community#122632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I requested to support page. None acknowledgement 3 weeks ago.
https://support.github.com/ticket/personal/0/2731070

We wish we need "Japanese tax 10% included" written for Invoice.
How we connecting for Github finance division?

answered and locked last inquiry. but no answer meaning.
https://github.com/orgs/community/discussions/121897

How we connecting for Github finance division?](https://github.com/orgs/community/discussions/121897)

**Comment by shoaib4567** [comment]
To address your issue with GitHub regarding the inclusion of "Japanese tax 10%" on your [invoices](https://www.genio.ac/mobile-invoices/), I recommend reaching out directly to GitHub's finance division. You can initiate contact by emailing their finance department with your request clearly stated. If you have a specific contact at GitHub, use their email address for direct communication. Additionally, submitting a ticket through GitHub Support, referencing your ticket number #2731070, can also help expedite your request. If you previously engaged in discussions on GitHub's community forums (https://github.com/orgs/community/discussions/121897), consider posting a follow-up there for further assistance. Persistence in follow-ups may be necessary if you haven't received a response, ensuring your requirement for Japanese tax inclusion is clearly communicated and addressed promptly.
